dense_vector/track.py

```diff
@@ -0,0 +1,31 @@
+import random
+
+
+def knn_param_source(track, params, **kwargs):
+    # choose a suitable index: if there is only one defined for this track
+    # choose that one, but let the user always override index
+    random.seed(31)
```

Comment:
Does this mean that the same sequence of random vectors will be produced across race runs? I think that's okay, just wanted to check.

Reply:
yes

Reply:
commented to that effect